Formula/gmp.rb

```diff
@@ -14,14 +14,34 @@ class Gmp < Formula
 
   uses_from_macos "m4" => :build
 
+  patch do
+    # Remove when upstream fix is released
+    # https://gmplib.org/list-archives/gmp-bugs/2020-July/004837.html
+    # arm64-darwin patch
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/6f75ed4d/gmp/6.2.0-arm.patch"
```

Comment:
Needs update due to https://github.com/Homebrew/formula-patches/pull/288 being rebased and merged.

```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/c53834b4/gmp/6.2.0-arm.patch"
```